src/interpreter.rs

```diff
@@ -1239,7 +1239,7 @@ impl<'source> Interpreter<'source> {
         fcn: &'source Expr<'source>,
         params: &'source Vec<Expr<'source>>,
     ) -> Result<Value> {
-        let fcn_rule = match self.lookup_function(fcn) {
+        let fcns_rule = match self.lookup_function(fcn) {
```

Comment:
`fcn_rules` would be a better name since there are many rules for the same function.

Reply:
Sounds good.